### PR TITLE
DODO-291 Add/batch normalization

### PR DESCRIPTION
**Pull request by eldesh**
Add a new BatchNormalization operator.
Only test mode of version 9 or higher is supported.

### CODE REVIEW

**Review by ishiy1993 — CHANGES_REQUESTED** [review]
Add tests for `opset_version = 14 and 15`.

**Comment by eldesh** [comment]
Existing operators (e.g,. https://github.com/Idein/onnion/blob/6174110f92c4fcc3de86e9a3ffc6941a610d9c74/runtime/tests/test_add.py) don't seem to test each version, so why is this necessary only for BatchNormalization?

File: runtime/onnion_runtime/batchnormalization.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class BatchNormalization:
+    def __init__(self, opset_version, **kwargs):
+        self.version = opset_version
+        self.epsilon = kwargs.get("epsilon", 1e-5)
+        if 14 <= self.version:
+            training_mode = kwargs.get("training_mode", 0)
+            assert training_mode == 0, "Non-zero values for training_mode are not supported"
+        assert 9 <= self.version, "Now, only opset_versions (>= 9) are supported"
```
Comment (ishiy1993) [inline runtime/onnion_runtime/batchnormalization.py:11]: Use `RunError`

**Comment by ishiy1993** [comment]
>Existing operators (e.g,. https://github.com/Idein/onnion/blob/6174110f92c4fcc3de86e9a3ffc6941a610d9c74/runtime/tests/test_add.py) don't seem to test each version, so why is this necessary only for BatchNormalization?

Because there is a new attribute `training_mode`.

**Review by ishiy1993 — APPROVED** [review]
LGTM

**Comment by eldesh** [comment]
:pray:
